packages/backend-plugin-api/src/services/system/types.ts

```diff
@@ -152,6 +154,7 @@ export interface PluginServiceFactoryConfig<
   TImpl extends TService,
   TDeps extends { [name in string]: ServiceRef<unknown> },
 > {
+  initialization?: 'always' | 'lazy';
```

Comment:
I know we have a documentation ticket, but if you're up for it, this really would benefit from a thorough doc comment